<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogcheck_out</class>
 <widget class="QDialog" name="Dialogcheck_out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>583</width>
    <height>468</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>583</width>
    <height>468</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>办理离店</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hotel.qrc">
    <normaloff>:/new/prefix1/Images/Common/LOGO.png</normaloff>:/new/prefix1/Images/Common/LOGO.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/Images/Common/check_out.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>客户姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_roomnumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roomnumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房费：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yes">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_account">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>结账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_no">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_extra">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>额外消费：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>总消费：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_price">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_extra">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_total">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_days">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>天数：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_days">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="hotel.qrc"/>
 </resources>
 <connections/>
</ui>
